lib/msf/core/exploit/format/php_payload_png.rb

```diff
@@ -30,6 +29,7 @@
 TODO: IDAT chunk     ✅                    ✅                  ✅                     ✅
 tEXt chunk           ✅                    ❌                  ❌                     ✅
 =end
+# This mixin module provides methods to inject persistent PHP payloads into a PNG file.
```

Comment:
This is already in the comment above. Is there a reason we are adding it in a second time?

Reply:
`rubocop` complains about it.
```
lib/msf/core/exploit/format/php_payload_png.rb:32:1: C: Style/Documentation: Missing top-level documentation comment for module Msf::Exploit::Format::PhpPayloadPng.
module Msf::Exploit::Format::PhpPayloadPng
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Reply:
Fixed in https://github.com/rapid7/metasploit-framework/pull/17507/commits/4af1feb6431407322ac8703351b7a67bf3bd8b6f